tests/redirects_tests/tests.py

```diff
@@ -63,6 +66,11 @@ def test_sites_not_installed(self):
             'django.contrib.sites is not installed.'
         )
         with self.assertRaisesMessage(ImproperlyConfigured, msg):
+            RedirectFallbackMiddleware(lambda req: HttpResponse())
+
+    def test_signature_deprecation(self):
```

Comment:
I put it there originally but contrib.redirects is not always installed, so I had to put it here.

Reply:
Yes I know, but it will easier to remove an entire `tests/deprecation/test_middleware_mixin.py` file when deprecation ends, even it know we have to add a separate test case with `@modify_settings(MIDDLEWARE={'append': 'django.contrib.redirects.middleware.RedirectFallbackMiddleware'})` :man_shrugging:

Reply:
The issue is I can't import RedirectFallbackMiddleware, because of an app_label issue. I'm not seeing how I can get round that?

Reply:
You can add `deprecation` to the `CONTRIB_TESTS_TO_APPS` in `runtests.py`, but it's probably not worth it.

Reply:
Nice... but more, it'll be left there forever after that. 

I thought the change here was OK, once I got over my initial disappointment 🙂 — The `RemovedIn...` means we won't forget it. 

I don't mind. Up to you.

Reply:
Scrub this. RedirectFallbackMiddleware calls super() 

Test already passed when I added it. (Forgot about that.) I will just remove it. 💃